src/account/components/AccountAvatar.js

```diff
@@ -8,17 +8,27 @@ const AccountAvatar = props => {
   const { user, showAlt } = props;
   const name = `${user.firstName} ${user.lastName}`;
 
+  const muiAvatarAllowedProps = [
+    'alt',
+    'children',
+    'classes',
+    'className',
+    'component',
+    'imgProps',
+    'sizes',
+    'src',
+    'srcSet',
+    'sx',
+    'variant',
```

Comment:
question: should include ID? or will that work without being in the list?

Reply:
@paulschreiber this will omit ID if it is sent, I checked and right now we are not passing ID to the AccountAvatar component, so I think we can leave it there and if is needed we can add it